.circleci/config.yml

```diff
@@ -19,6 +19,23 @@ jobs:
           name: lint
           command: make lint
 
+  build-docs:
+    working_directory: ~/repo
+
+    docker:
+      - image: python:3.8-alpine
+
+    steps:
+      - checkout
+
+      - run:
+          name: Install prerequisites
+          command: pip install -r docs/requirements-doc.txt
+
+      - run:
+          name: Build docs
+          command: python3 -m sphinx -M html docs docs/_build
```

Comment:
Could you store `docs/_build` as artifacts, so we can actually check the output in PR?